backend/app/platforms/router.py

```diff
@@ -34,6 +35,7 @@ def get_platform_services(
 
 @router.get(
     "/{platform_id}/services/{service_id}",
+    dependencies=[Depends(only_for_admin)],
```

Comment:
I am not the biggest fan of having these in place for Get calls, because that also blocks the frontend UI.
It should not be restricted to admins to show the result, it should be to edit it ofcourse. But in data outputs for example we want to show the full s3 bucket arns also to non admins

Reply:
Could be that this is a remnant of the other mismerged things. But I would remove it :)

Reply:
This was indeed a remnant of the merges, I removed it.